idbmms-connector/src/main/java/com/sequenceiq/cloudbreak/idbmms/config/IdbmmsClientConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.sequenceiq.cloudbreak.idbmms.config;
+
+import org.springframework.context.annotation.Configuration;
+
+import io.netty.util.internal.StringUtil;
+
+@Configuration
+public class IdbmmsClientConfig {
+
+    private final String callingServiceName;
```

Comment:
Shouldn't this be a property value like the following?
```
@Value("${altus.ums.caller:cloudbreak}")

Reply:
I've got it fixed, based on suggestion:
```
@Autowired
public IdbmmsClientConfig(@Value("${altus.idbmms.caller:cloudbreak}") String callingServiceName) {
   this.callingServiceName = callingServiceName;
}
```
Thx, @biharitomi